az-core/src/main/java/azkaban/Constants.java

```diff
@@ -182,6 +184,8 @@
     public static final String METRICS_SERVER_URL = "azkaban.metrics.server.url";
 
     public static final String IS_METRICS_ENABLED = "azkaban.is.metrics.enabled";
+    public static final String MIN_AGE_FOR_CLASSIFYING_A_JOB_AGED_MINUTES = "azkaban.metrics"
+        + ".min_age_for_classifying_a_job_aged_minutes";
```

Comment:
Please consider using FLOW instead of JOB in the name as the new metric detects delayed flow executions not job executions.

Reply:
Done